corehq/motech/fhir/tests/test_fhir_views.py

```diff
@@ -57,13 +59,28 @@ def _get_response(self, path):
             HTTP_AUTHORIZATION=f'ApiKey {USERNAME}:{self.api_key.key}'
         )
 
+    def assertHQStandard404Page(self, response):
```

Comment:
Should we support only json request?

Reply:
Yes! And now that I've written a client for this API, I can confirm how annoying it is to get HTML responses!

I'll save this for a separate PR though, cos it's outside the scope of enabling API key auth.

Reply:
Cool. I was hoping that it would make some tests added, redundant. Fine to keep it for later.